src/Service/AbstractJiraService.php

```diff
@@ -246,6 +252,79 @@ public function getAllCustomers()
         return $accounts;
     }
 
+    public function getAccount($accountId)
+    {
+        return $this->get('/rest/tempo-accounts/1/account/'.$accountId.'/');
+    }
+
+    public function getRateTableByAccount($accountId)
+    {
+        return $this->get('/rest/tempo-accounts/1/ratetable', [
+            'scopeId' => ''.$accountId,
```

Comment:
Do you mean `(string)$accountId`? Is it necessary to convert to a string?

Reply:
No, it is not